### PR TITLE
Update Contact Manager markdown file

### PR DESCRIPTION
**Pull request by applegrain**
- removed reference to Sass and Haml in the intro 
- changed hash syntax to the newer one to make it more consistent
- fixed backtick formatting, some single quotes were backticks which made the formatting look off
- update EI6 Auth section

### CODE REVIEW

File: source/projects/contact_manager.markdown [hunk 1]
```diff
@@ -2276,8 +2275,8 @@ To get this to pass I changed my create method to this:
 ```ruby
 def create
   data = request.env['omniauth.auth']
-  user = User.where(:provider => data['provider'], :uid => data['uid'], :name => data['info']['name']).first_or_create
-  render :nothing => true
+  user = User.where(providerL data['provider'], uid: data['uid'], name: data['info']['name']).first_or_create
```
Comment (jmejia) [inline source/projects/contact_manager.markdown]: @applegrain `provider:` instead of `providerL` :cop: 

Overall this looks really good :sparkles:

**Comment by jmejia** [comment]
@applegrain Thanks the PR! There's just one minor change and I will merge this in :+1:

File: source/projects/contact_manager.markdown (hunk 1 above)
Comment (applegrain) [inline source/projects/contact_manager.markdown]: Oops, it went a little bit too fast there! Thanks :)

**Comment by jmejia** [comment]
@applegrain Sorry, one more thing. Which version of Rails are you using? The tutorial specifies 4.1.6. If your's is different, can you update the top of section I0?

**Pull request merged**